templates/home.html

```diff
@@ -0,0 +1,4 @@
+{% extends "base.html" %}
+{% block content %}
+<h3>Home</h3>
+{% endblock %}
```

Comment:
The **home** page inherits from our base template and adds some custom placeholder text.